spec/integration/client_spec.rb

```diff
@@ -38,6 +36,10 @@
   let(:subscriber) { Routemaster::Models::Subscriber.find(uuid) }
   let(:topic) { Routemaster::Models::Topic.find('widgets') }
 
+  it 'populates logins from $ROUTEMASTER_CLIENTS at boot' do
```

Comment:
That's not really an integration test, since you're checking implementation details (what is stored in Redis) instead of behaviour (can I use the API with that key).

Would you mind changing the test accordingly?